File: Java_Programming/Competitive/PrimeNumber.java
class PrimeNumber
{
    public static void main (String A[])
    {
    boolean bRet = false;
    Logic lobj = new Logic();

    bRet = lobj.CheckPrime(11);
    
    if(bRet==true)
    {
        System.out.println("Number is Prime");
    }
    else
    {
        System.out.println("Number is not Prime");
    }

    }

}

class Logic
{
    boolean CheckPrime(int iNo)
    {
        int iCnt=0;
        int iFactor=0;
        for(iCnt=1; iCnt<=iNo; iCnt++)
        {
            iFactor++;  

        }
        if(iFactor==2)
        {
            return true;
        }
        else
        {
            return false;

        }

    }
}

File: Python/Competetive/PrimeNumber.py
def prime(Value):
    
    for iCnt in range(2,Value):
        if(Value%iCnt==0):
            return True
            break
        else:
            return False

def main():
    print("Enter The Number: ")
    No= int (input())

    bRet=prime(No)
    if(bRet==True):
        print(No," Not A Prime Number")

    else:
        print(No," is A Prime Number")
   


if __name__=="__main__":
    main()